### DISCUSSION TITLE
cant build images

### DISCUSSION THREAD

**Discussion by Masber** [goreleaser/goreleaser#5387] [Q&A]
Dear goreleaset community,

I am new to gorelease and go in general, I have a go project integrated with goreleaser which fails while building the docker image? my machine is a macbook pro m3 and I am getting the error below.

These are the instructions I use

> docker build failed: failed to build  exit status 125: unknown flag: --load

Any idea of the workaround for this issue?

**Comment by caarlos0** [comment]
what version of docker/docker buildx?

the ones i have here fwiw:

```sh
❯ docker buildx version
github.com/docker/buildx v0.18.0

❯ docker --version
Docker version 27.3.1, build v27.3.1
```

you can also check `docker buildx build --help` and check if `--load` shows up there.
